sdb/command.py

```diff
@@ -367,8 +367,8 @@ def _call(self, objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
                     #
                     err_msg = f"invalid memory access: {str(err)}"
                 else:
-                    err_msg = "invalid memory access while handling object "
-                    err_msg += "at address {hex(obj.address_of_().value_())}"
+                    err_msg = "invalid memory access while handling object at address %s" % (
+                        hex(obj.address_of_().value_()))
```

Comment:
I think we're preferring to use "f-strings" as we do a couple lines above
```suggestion
                        f"at address {hex(obj.address_of_().value_())}"
```